docs/website/docs/dlt-ecosystem/destinations/staging.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Staging
+description: Configure an s3 or gcs bucket for staging before copying into the destination
+keywords: [staging, destination]
+---
+
+# Staging
+
+dlt supports a staging location for some destinations. Currently it is possible to copy files from a s3 bucket into redshift and from a gcs bucket into bigquery. dlt will automatically select an appropriate loader file format for the staging files. For this to work you have to set the staging argument of the pipeline to filesystem and provide both the credentials for the staging and the destination.
+
+## Redshift and s3
+
+```python
```

Comment:
same for redshift. but in case of redshift we must also explain the config options
1. forward credentials - what happens and how it works
2. if you do not forward (which we recommend - how you set up a cluster) - I can help with this!
3. which file formats are supported. I think this should be somewhere at the top of the doc: if you want to load parquet or jsonl you must use staging! otherwise it is INSERT VALUES

Reply:
I have made those changes for now and will try to figure out authentication via the cluster.